consensus/consensus-types/src/block.rs

```diff
@@ -241,6 +244,17 @@ impl Block {
                 self.timestamp_usecs() > parent.timestamp_usecs(),
                 "Blocks must have strictly increasing timestamps"
             );
+
+            let current_ts = SystemTime::now()
+                .duration_since(UNIX_EPOCH)
+                .expect("Current time is before the UNIX_EPOCH!");
```

Comment:
note: we're trying to get rid of any expect/unwrap/unimplemented/unreachable in functions that return a `Result`. I was trying to think of cases where it wouldn't make sense to not panic, I think this is a good example of using `expect()` in a legitimate way. The only problem is that it'll make our lint complain once we have that, so perhaps creating a helper function called `get_current_time()` or something like that would hide the expect : o

Reply:
Actually it exists already, I should have checked:
https://github.com/libra/libra/blob/f131132465a3af7f34820e4f1498f5142a7fe1bf/consensus/src/util/time_service.rs#L131-L135

But even if the expect is hidden, if `duration_since_epoch()` panics, wouldn't it still cause it to panic? So this is really just to exploit the fact the linter is dumb?